### PR TITLE
Create sh_hsk_what-is-a-stake-pool

### PR DESCRIPTION
**Pull request by nahern**
## What? 

new file to describe stake pools for About section in Haskell testnet website.

## Why? (optional)

new section being added.

## How can this be tested? (optional)

Describe how this change can be tested

## Screenshots (optional)

Upload some screenshots of the changes

### CODE REVIEW

File: resources/content/articles/en/sh_hsk_what-is-a-stake-pool [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: What is a stake pool
```
Comment (jabbacakes) [inline resources/content/articles/en/sh_hsk_what-is-a-stake-pool:2]:
```suggestion
title: What is a stake pool?
```
We have a question mark in the nav heading for 'What is a Cardano node?' - so we should have one here for consistency.

File: resources/content/articles/en/sh_hsk_what-is-a-stake-pool [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+description: About
+parent: 2020-05-04_05-00-00_about-en.md
+order: 2
+last_updated: "2020-05-22T09:00:00+01:00"
+---
+## What is a stake pool
```
Comment (jabbacakes) [inline resources/content/articles/en/sh_hsk_what-is-a-stake-pool:8]:
```suggestion
## What is a stake pool?
```

File: resources/content/articles/en/sh_hsk_what-is-a-stake-pool [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Once a stake pool is set up and ready for use, it must be registered on the blockchain. All stake pools must be registered and registration information is recorded on-chain. The stake pool owners may also  choose to register the pool with one or more stake pool registries. These registries are maintained off-chain and are used to compile lists that distinguish stake pools within the Daedalus wallet interface and other tools/web sites. However, it is important to note that being part of a registry is not mandatory, and anyone with enough ada to pay the transaction fee can set up a stake pool, even if they are not listed in any registry. Ada holders may also choose to subscribe to the registries of their choice, or to no registry at all.  An ada holder can then go to Daedalus and select the stake pool to which they want to delegate their stake, in the Delegation tab. That delegation action is recorded on the Cardano blockchain and assuming  it is recorded sufficiently early in the epoch, may influence the slot leader selection process in the next epoch and beyond.
```
Comment (jabbacakes) [inline resources/content/articles/en/sh_hsk_what-is-a-stake-pool:15]:
```suggestion
Once a stake pool is set up and ready for use, it must be registered on the blockchain. All stake pools must be registered and registration information is recorded on-chain. The stake pool owners may also choose to register the pool with one or more stake pool registries. These registries are maintained off-chain and are used to compile lists that distinguish stake pools within the Daedalus wallet interface and other tools and web sites. However, it is important to note that being part of a registry is not mandatory, and anyone with enough ada to pay the transaction fee can set up a stake pool, even if they are not listed in any registry. Ada holders may also choose to subscribe to the registries of their choice, or to no registry at all.  An ada holder can then go to Daedalus and select the stake pool to which they want to delegate their stake, in the Delegation tab. That delegation action is recorded on the Cardano blockchain and assuming it is recorded sufficiently early in the epoch, may influence the slot leader selection process in the next epoch and beyond.
```

Few double spaces.

**Comment by jabbacakes** [comment]
@lewnelson Can you confirm that this file name `sh_hsk_what-is-a-stake-pool.md` won't break everything? Or will the lack of prepended date make Netlify freak out?

File: resources/content/articles/en/sh_hsk_what-is-a-stake-pool.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: What is a stake pool?
+description: About
+parent: 2020-05-04_05-00-00_about-en.md
```
Comment (lewnelson) [inline resources/content/articles/en/sh_hsk_what-is-a-stake-pool.md:4]: This needs to be `2020-05-04_05-00-00_about-en`

**Pull request merged**